src/basic/io-util.h

```diff
@@ -74,3 +74,19 @@ static inline bool FILE_SIZE_VALID_OR_INFINITY(uint64_t l) {
 
 char *set_iovec_string_field(struct iovec *iovec, size_t *n_iovec, const char *field, const char *value);
 char *set_iovec_string_field_free(struct iovec *iovec, size_t *n_iovec, const char *field, char *value);
+
+struct iovec_wrapper {
+        struct iovec *iovec;
+        size_t count;
+        size_t size_bytes;
+};
+
+#define IOVW_INIT { .iovec = NULL, .count = 0, .size_bytes = 0 }
```

Comment:
Please drop this define, it obfuscates more than adds any value. Just say `iovw = {}` everywhere.